src/css/SocialsIcon.module.css

```diff
@@ -3,17 +3,24 @@
   flex-direction: row;
   align-items: center;
   justify-content: space-between;
-  width: 129px;
+  width: max-content;
+}
+
+.iconLink {
+  margin-right: 0.75rem;
 }
 
 .iconLink:hover {
-  opacity: 0.95;
+  opacity: 0.9;
 }
 
 .img {
   opacity: 1;
+  width: 2.5rem;
+  height: 2.5rem;
+  filter: invert(12%) sepia(41%) saturate(5654%) hue-rotate(226deg) brightness(89%) contrast(90%);
 }
 
 .img:hover {
```

Comment:
So this is entirely not on you, but having a `:hover` state for both the image and the parent `<a>` link is really redundant and confusing, since the hierarchy is `<a.iconLink><img.img>`. I'd actually recommend we take out the `.img:hover` styling entirely, and compensate by changing `.iconLink:hover`'s opacity to `0.81` to compensate (before adjusting it again like my previous comment suggests).

Reply:
I totally agree with this, it's unneccesary to have hover for both `img.img` and `a.iconLink`. I will play around with this to see what works best. Thanks for bringing this up 👍🙇‍♀️